src-electron/db/db-mapping.js

```diff
@@ -155,6 +157,7 @@ exports.map = {
       argIsArray: dbApi.fromDbBool(x.ARG_IS_ARRAY),
       argPresentIf: x.ARG_PRESENT_IF,
       argCountArg: x.ARG_COUNT_ARG,
+      argIsNullable: x.ARG_IS_NULLABLE,
```

Comment:
This is true if any arg is nullable?

Not sure we will need this, but ok.

Reply:
This is part of a join, and it's note called "IS_NULLABLE", but "ARG_IS_NULLABLE" to bypass possible naming conflicts.